.github/workflows/coverage-pr.yml

```diff
@@ -1,12 +1,15 @@
 name: Check coverage for PR
 
 on:
-  pull_request_target:
+  pull_request:
```

Comment:
What was the reason to revert back to pull_request? It can break dependabot prs

Reply:
It was [this](https://github.com/fingerprintjs/fingerprintjs-pro-cloudflare-worker/commit/cb5b34a1) commit. I don't remember the reason. Right now, all dependabot PRs are created based on the `develop` branch, so I don't think it will be an issue. Regardless, I can revert it back to `pull_request_target`

Reply:
OK, let's keep it and decide what to do later.